debian/control

```diff
@@ -6,22 +5,30 @@ Maintainer: Go Compiler Team <pkg-golang-devel@lists.alioth.debian.org>
 Uploaders: Michael Stapelberg <stapelberg@debian.org>,
            Paul Tagliamonte <paultag@debian.org>,
            Tianon Gravi <admwiggin@gmail.com>
-Vcs-Browser: http://anonscm.debian.org/gitweb/?p=pkg-golang/golang.git
+Vcs-Browser: https://anonscm.debian.org/cgit/pkg-golang/golang.git
 Vcs-Git: git://anonscm.debian.org/pkg-golang/golang.git
-Build-Depends: debhelper (>= 7.4.10), bison, ed, mawk | awk, perl, netbase
+Build-Depends: bison,
+               debhelper (>= 7.4.10),
+               ed,
+               golang (>= 1:1.4~) | gccgo-5,
+               mawk | awk,
+               netbase,
+               perl
 Build-Depends-Indep: po-debconf
 Standards-Version: 3.9.6
-Homepage: http://golang.org/
+Homepage: https://golang.org
 
 Package: golang-go
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends},
-	 golang-src (>= ${source:Version}),
-	 golang-go-linux-${go:Hostarch} (= ${source:Version})
-Replaces: golang-weekly-go, golang-weekly-tools, golang-tools
-Conflicts: golang-weekly-go, golang-weekly-tools, golang-tools
-Provides: go-compiler, golang-weekly-go, golang-weekly-tools, golang-tools
-Suggests: bzr, git, golang-go.tools, mercurial, subversion
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
+Depends: golang-pkg (= ${source:Version}),
+         golang-src (>= ${source:Version}),
```

Comment:
We did the split to golang-go-HOST-ARCH only because the package was like 500 MB in total, and that’s too much to tell people to just deal with it :). If this is no longer necessary, I’m happy to consolidate these back together.

With regards to merging the other packages, I’m fine with that as well. At some point, golang-src was only used as documentation (accessed through godoc), but I see that golang-go has a dependency on it.